project/ISSUES.md

```diff
@@ -0,0 +1,171 @@
+  message and/or commentary
+* `status/WIP` - this issue or PR is currently being worked on.  It should not
+  be merged (or closed) without pinging the owner/submitter.
+* `status/review` - issue is resolved or PR is complete and needs review
+
+Note that there are no `status/complete`, `status/wontfix`, `status/duplicate`,
+or other terminal status indicators.  This is intentional to keep the tail end
+of bookkeeping from getting onerous.  These sorts of terminal states and their
+justifications should be indicated in the closing comment by the issue closer.
+
+Focus Labels
+------------
+
+The 'focus' label is used for tagging issues and pull requests to a particular
+focus area.  This allows for easy searching within that particular domain, as
+well as more organized release notes.  Some examples, not-exhaustive, are here:
+
+* `focus/API`
+* `focus/cleanup`
+* `focus/jobs`
+* `focus/tests`
+* `focus/GIEs`
+* `focus/toolshed`
+* `focus/UI-UX`
+* `focus/workflows`
+
+This list will definitely grow over time.
+
+Other Useful Labels
+-------------------
+
+While the three labels sets indicating status, focus, and type are required
+there are several other labels that are be useful and/or have special purpose.
+
+* `Procedures` is a special tag that indicates that the issue is related to
+  project governance, and it overrides the need for the trio of
+  class/status/focus tags, and these are never auto-flagged for triage.
+
+* `Planning` is also a special tag that indicates the issue is related to
+  larger-scale issue planning.  These issues are typically meta-issues
+  containing checklists and references to other issues which are subcomponents
+  and stepping stones necessary for issue resolution.  These *can* utilize the
+  `focus/*` tags but are not required to.  Status and type make little sense
+  here.
+
+* `Roadmap` is a reserved tag for the primary project roadmap.  This is a
+  meta-issue that is not expected to be completed, but rather serves as an
+  entrypoint to the high level development of the project.
+
+* `beginner-friendly` can be used to indicate a nice entry-level issue that
+  only requires limited understanding of the larger Galaxy framework and
+  ecosystem.  This is useful for encouraging new contributors.
+
+
+The Roadmap
+===========
+
+We will maintain a single `Roadmap` tagged meta-issue which will describe (at a
+very high level) the *current* major areas of focus for the project.  This is
+similar to our PRIORITIES 2014/15 cards on Trello.  Using [Task
+Lists](https://github.com/blog/1375-task-lists-in-gfm-issues-pulls-comments),
+this issue will link to sub-issues which will go into much more detail, might
+have its own checklists to even more subcomponent cards, and so on.  
+
+This `Roadmap` issue will be assigned to every release milestone, forcing
+periodic review of the roadmap.
+
+To prevent the roadmap from being tied completely to github, and to facilitate
```

Comment:
Basically what I had in mind was every time the roadmap was updated (the live GitHub issue, which we will handle exactly like you're suggesting) we copy/paste that into roadmap.md to have a historical record of it regardless of where the codebase and/or issue tracking ends up moving next.  Maybe this isn't worth the effort though.